docs/README.md

```diff
@@ -15,9 +15,9 @@
     <a href="https://www.vidurjobs.xyz"><strong>Learn more »</strong></a>
```

Comment:
change this too

Reply:
@amandesai01 okay but I need link for this change